src/features/game/dialogue/GameDialogueManager.ts

```diff
@@ -66,11 +66,12 @@ export default class DialogueManager {
       });
   }
 
-  private async showNextLine(resolve: () => void) {
+  public async showNextLine(resolve: () => void) {
```

Comment:
Is there a need to change this from private to public?

Reply:
The showNextLine method is used in GameGlobalAPI and GameQuizManager for resolve the hidden dialogue object before the start of the quiz.

Reply:
Resolving the conversation for now as changing access is needed for `GameGlobalAPI`. (GameGlobalAPI is a separate issue that is out of scope for this PR 😓)